doc/source/dev/contributor/quickstart_docker.rst

```diff
@@ -0,0 +1,177 @@
+ software that includes everything needed to run an application: code, runtime,
+ system tools, system libraries and settings.
+ Container images become containers at runtime and in the case of Docker
+ containers - images become containers when they run on Docker Engine.
+ Available for both Linux and Windows-based applications, containerized
+ software will always run the same, regardless of the infrastructure.
+
+Docker makes setting up a development environment easy and reliable: we
+provide a Docker image with suitable compilers already installed; you
+use the Docker engine to execute the image as a container, add the latest
+development version of SciPy and its build-time dependencies, and build
+SciPy.
+There are Docker hosts for several OS’s including
+macOS, Linux, and Windows; please follow the appropriate
+installation instructions for your operating system at `docs.docker.com`_.
+
+.. note::
+
+   If you have a version of an operating system that doesn't meet the
+   requirements of Docker Desktop, such as Windows 10 Home,
+   try `Docker Toolbox`_ .
+
+Cloning SciPy
+-------------
+
+Before starting SciPy's docker container, you should create a copy of the
+SciPy source code on your computer. That way, you'll be able to access the
+same files both from your native operating system and within the container.
+
+*Note: below we will use* terminal window *as a
+collective term that includes the Windows Command Prompt.*
+
+#. Browse to the `SciPy repository on GitHub`_ and `create your own fork`_.
+   You'll need to create a GitHub account if you don’t
+   already have one.
+#. Browse to your fork. Your fork will have a URL like
+   https://github.com/andyfaff/scipy, except with your GitHub username
+   in place of "andyfaff".
+#. Click the big, green "Clone or download" button, and copy the ".git"
+   URL to the clipboard. The URL will be the same as your fork’s url,
+   except it will end in ".git".
+#. Create a folder for the SciPy source code in a convenient place on
+   your computer. `Navigate`_ to it in the terminal window.
+#. Enter the command ``git clone`` followed by your fork’s .git URL.
+   Note that this creates in the terminal’s working directory a
+   ``scipy`` folder containing the SciPy source code. This assumes that
+   you have a ``git`` command line client that is available on your
+   PATH; if not, you can follow these `instructions to install a git client`_.
+
+Starting Docker
+---------------
+
+Instructions for getting started with Docker can be found `here`_. After
+ensuring that Docker is working correctly, follow the instructions below to
+start a Docker container for SciPy development. You'll follow the same
+instructions each time you want to start the container, as changes made to a
+container do not persist after you close it.
+
+#. In a terminal window, change the directory (using the ``cd`` command)
+   to the folder that contains the SciPy git repository.
```

Comment:
> to the root folder of the SciPy git repository (holding the main `setup.py` file)

Reply:
OK, that is probably more clear. I just copy pasted from wiki.